spec/models/user_spec.rb

```diff
@@ -17,6 +17,22 @@
     end.to change(ActionMailer::Base.deliveries, :count).by(1)
   end
 
+  describe 'password validations' do
+    it 'allows long phrases that contain common words' do
+      user = create(:user)
+      user.password = 'a long password'
+
+      expect(user.valid?).to eq true
+    end
+
+    it 'allows unconfirmed visitors' do
+      user = create(:user, :unconfirmed)
+      user.password = 'a long password'
+
+      expect(user.valid?).to eq true
+    end
```

Comment:
heh, I dropped the test for invalid per this thread: https://github.com/18F/identity-idp/pull/536#discussion_r82108232